dashboards/lodestar_vm_host.json

```diff
@@ -1,9 +1,49 @@
 {
+  "__inputs": [
```

Comment:
This was added because I used the `Export for sharing externally` option which should be correct in my opinion.
Not sure about the other additions, probably related to a newer grafana version, I made sure that the dashboards is not changed anywhere else, just added one panel for approximate db size duration.

Would make sense if someone reviewing quickly imports this as well to double check it